widgets/GCSMultiFiles-batchsink.json

```diff
@@ -0,0 +1,136 @@
+      "properties" : [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName"
+        }
+      ]
+    },
+    {
+      "label" : "Service Account and Project Information",
+      "properties" : [
+        {
+          "widget-type": "textbox",
+          "label": "Service Account File Path",
+          "name": "serviceFilePath",
+          "widget-attributes" : {
+            "placeholder": "Path to the service account file on local file system"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Project ID",
+          "name": "project",
+          "widget-attributes" : {
+            "placeholder": "The project id of the Cloud Storage bucket"
+          }
+        }
+      ]
+    },
+    {
+      "label": "File Configuration",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Base Path",
+          "name": "path"
+        },
+        {
+          "widget-type": "prefix",
+          "label": "GCS Folder Name Prefix",
+          "name": "prefix"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Path Suffix",
+          "name": "suffix",
+          "widget-attributes": {
+            "default": "yyyy-MM-dd-HH-mm"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Bucket Name",
+          "name": "bucket"
+        },
+        {
+          "widget-type": "select",
+          "label": "Compression Codec",
+          "name": "codec",
+          "widget-attributes": {
+            "values": [
+              "None",
+              "Snappy",
+              "Deflate",
+              "GZip",
+              "LZO"
+            ],
+            "default": "None"
+          }
+        }
+      ]
+    },
+    {
+      "label": "Multi Format Google Cloud Storage Sink Configuration",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Split Field",
+          "name": "splitField",
+          "widget-attributes": {
+            "default": "tablename"
+          }
+        },
+        {
+          "widget-type": "select",
+          "label": "File Output Format",
+          "name": "outputFormat",
+          "widget-attributes": {
+            "values": [
+              "text",
+              "avro",
+              "orc",
+              "parquet"
+            ],
+            "default": "text"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Field Delimiter",
+          "name": "delimiter",
+          "widget-attributes": {
+            "default": "\t"
+          }
+        }
+      ]
+    }
+  ],
+  "outputs": [
+    {
+      "name": "schema",
+      "widget-type": "schema",
+      "widget-attributes": {
+        "schema-types": [
+          "boolean",
+          "int",
+          "long",
+          "float",
+          "double",
+          "bytes",
```

Comment:
Might want to include Timestamp, datetime

Reply:
Will open additional PR for that one